RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -700,6 +700,14 @@ void finalizeMaximize(boolean speculative)
 		// blocks first hit, but doesn't stack with blood bubble
 		addBonusToMaximize($item[Eight Days a Week Pill Keeper], 100);
 	}
+	if (possessEquipment($item[June cleaver])) {
+		addBonusToMaximize($item[June cleaver], 100); // We want to ramp this up and the NCs are nice as well
```

Comment:
should this skip the bonus when no more NCs can be reached and turngen is over? would probably still be equipped but could avoid taking over off hands when at end of day and something else is imposed in main hand, condition to skip could be like

>if(get_property("_juneCleaverFightsLeft").to_int() < my_adventures()*1.1 && (fullness_limit() > 0 || inebriety_limit() > 0) && consumptionProgress() >= 1)

Reply:
Quark has a good idea. I'm also wondering if we should only give a bonus while we still have skips available. Haven't looked at how valuable the average cleaver NC is

Reply:
good idea